public/app/plugins/datasource/loki/components/useLokiSyntaxAndLabels.ts

```diff
@@ -45,7 +43,6 @@ const useLokiSyntax = (languageProvider: LokiLanguageProvider, languageProviderI
   useEffect(() => {
```

Comment:
This is currently not needed for PrismSlate plugin, but we are using syntax and hasSyntaxLoaded for Cascader Button - to set it's text and at other places. Therefore it needs more thinking regarding if we want to keep this or refactor this part of the codebase so it is not needed.

Reply:
hmm so I think we can just remove it and use `languageProvider.getSyntax()` and `isSyntaxReady`  will just be true cause there is not specific async init anyway right? Or is there some other caveat here?